### ISSUE TITLE
New Site Setup Config Import Fail Due To Shortcut Link Entity Existing

### ISSUE THREAD

**Issue by stephsem23** [acquia/blt#1948]
My system information:
* Operating system type: macOS Sierra
* Operating system version: 10.12.4
* BLT version: 8.8.4

Output of `blt doctor`:
```
blt > doctor:
array_replace_recursive(): Argument #2 is not an array BltDoctorCommand.php:141                                                                                                         [warning]
Changed current directory to /home/vagrant/.composer
+----------------------+----------------------------------------------------------------------------------+
| Property             | Value                                                                            |
+----------------------+----------------------------------------------------------------------------------+
| %paths               |                                                                                  |
|  - %root             | /var/www/octagon/docroot                                                         |
|  - %site             | sites/default                                                                    |
|  - %modules          | modules                                                                          |
|  - %themes           | themes                                                                           |
|  - %config-sync      | /var/www/octagon/config/default                                                  |
|  - %files            | sites/default/files                                                              |
|  - %private          | /var/www/octagon/files-private                                                   |
|  - %temp             | /tmp                                                                             |
| admin-theme          | seven                                                                            |
| blt-version          | 8.8.4                                                                            |
| bootstrap            | Successful                                                                       |
| config-sync          | /var/www/octagon/config/default                                                  |
| db-driver            | mysql                                                                            |
| db-hostname          | localhost                                                                        |
| db-name              | octagon_default                                                                  |
| db-password          | admin                                                                            |
| db-port              | 3306                                                                             |
| db-status            | Connected                                                                        |
| db-username          | admin                                                                            |
| drupal-settings-file | sites/default/settings.php                                                       |
| drupal-version       | 8.3.7                                                                            |
| drush-alias-files    | /var/www/octagon/docroot/../drush/site-aliases/aliases.drushrc.php               |
|                      | /var/www/octagon/docroot/../drush/site-aliases/example.acsf.aliases.drushrc.php  |
|                      | /var/www/octagon/docroot/../drush/site-aliases/example.local.aliases.drushrc.php |
| drush-conf           | /var/www/octagon/drush/drushrc.php                                               |
| drush-script         | /var/www/octagon/vendor/drush/drush/drush.php                                    |
| drush-temp           | /tmp                                                                             |
| drush-version        | 8.1.10                                                                           |
| files                | sites/default/files                                                              |
| install-profile      | standard                                                                         |
| local-settings       | /var/www/octagon/docroot/sites/default/settings/local.settings.php               |
| modules              | modules                                                                          |
| php-bin              | /usr/bin/php                                                                     |
| php-conf             | /etc/php/5.6/cli/php.ini                                                         |
| php-os               | Linux                                                                            |
| private              | /var/www/octagon/files-private                                                   |
| root                 | /var/www/octagon/docroot                                                         |
| site                 | sites/default                                                                    |
| temp                 | /tmp                                                                             |
| theme                | meyers                                                                           |
| themes               | themes                                                                           |
| uri                  | default                                                                          |
| user                 |                                                                                  |
+----------------------+----------------------------------------------------------------------------------+
+--------------------------+---------------------------------------------------------------------------------+
| Check                    | Outcome                                                                         |
+--------------------------+---------------------------------------------------------------------------------+
| checkDocrootExists       | Found docroot.                                                                  |
| checkCoreExists          | Drupal core exists                                                              |
| checkSettingsFile        | BLT settings are included in settings file.                                     |
| checkLocalSettingsFile   | Found your local settings file.                                                 |
| checkLocalDrushFile      | Local drushrc file does not exist.                                              |
|                          | Create /var/www/octagon/docroot/sites/default/local.drushrc.php.                |
|                          | Run `blt setup:drush:settings` to generate it automatically, or run `blt setup` |
|                          | to run the entire setup process.                                                |
| checkFileSystem:%files   | Public files directory is writable.                                             |
| checkFileSystem:%private | Private files directory is writable.                                            |
| checkFileSystem:%temp    | Temporary files directory is writable.                                          |
| checkDbConnection        | Connected to database.                                                          |
| checkDrupalBootstrapped  | Bootstrapped Drupal via drush.                                                  |
| checkDrupalInstalled     | Drupal is installed.                                                            |
| checkCaching:page        | Drupal cache is disabled.                                                       |
| checkCaching:css         | CSS preprocessing is disabled.                                                  |
| checkCaching:js          | JS preprocessing is disabled.                                                   |
| checkNvmExists           | NVM does not exist.                                                             |
|                          |                                                                                 |
|                          | It is recommended that you use NVM to manage multiple versions of NodeJS on one |
|                          | machine.                                                                        |
|                          | Instructions for installing NVM can be found at:                                |
|                          |   https://github.com/creationix/nvm#installation                                |
| checkComposer:require    | acquia/blt is in composer.json's require object.                                |
| checkComposer:plugins    | hirak/prestissimo plugin for composer is installed.                             |
| checkBehat:exists        | Behat local settings file exists.                                               |
| checkBehat:uri           | base_url in tests/behat/local.yml does not match the site URI.                  |
|                          |   Behat base_url is set to "http://octagon.local".                              |
|                          |   Drush site URI is set to default.                                             |
| checkProjectYml:keys     | project.yml has no deprecated keys.                                             |
| checkAcsfConfig          | BLT settings are included in your pre-settings-php include.                     |
| checkPhpDateTimezone     | PHP setting for date.timezone is correctly set                                  |
+--------------------------+---------------------------------------------------------------------------------+

BUILD FINISHED; 15.8896 seconds

```

While setting up a new multisite with the following setup command:
```
blt octa:setup:new-site -Dsite.name=niche
```

At this step:
```
blt > drupal:install:
```

I get the following output:
```
blt > drupal:install:
     [echo] Installing Drupal...
You are about to DROP all tables in your 'octagon_niche' database. Do you want to continue? (y/n): y
Starting Drupal installation. This takes a while. Consider using the --notify global option.   
Installation complete.                                                                                                                                                                       [ok]
Do you want to update uuid key in system.site config? (y/n): y
 Collection  Config                                                                            Operation 
             block_content.type.basic                                                          create    
             comment.type.comment                                                              create    
             core.date_format.fallback                                                         create    
             core.date_format.html_date                                                        create    
             core.date_format.html_datetime                                                    create    
             core.date_format.html_month                                                       create    
             core.date_format.html_time                                                        create    
             core.date_format.html_week                                                        create    
             core.date_format.html_year                                                        create    
             core.date_format.html_yearless_date                                               create    
             core.date_format.long                                                             create    
             core.date_format.medium                                                           create    
             core.date_format.short                                                            create    
             core.entity_view_mode.block_content.full                                          create    
             core.entity_view_mode.comment.full                                                create    
             image.style.large                                                                 create    
             image.style.medium                                                                create    
             image.style.thumbnail                                                             create    
             block.block.meyers_page_title                                                     create    
             block.block.meyers_local_tasks                                                    create    
             block.block.meyers_local_actions                                                  create    
             block.block.meyers_help                                                           create    
             meyers.settings                                                                   create    
             field.storage.node.field_image                                                    create    
             field.storage.node.comment                                                        create    
             core.entity_view_mode.node.teaser                                                 create    
             core.entity_view_mode.node.search_result                                          create    
             core.entity_view_mode.node.search_index                                           create    
             core.entity_view_mode.node.rss                                                    create    
             core.entity_view_mode.node.full                                                   create    
             node.type.article                                                                 create    
             field.field.node.article.field_image                                              create    
             field.field.node.article.comment                                                  create    
             node.type.octagon_basic_page                                                      create    
             core.base_field_override.node.octagon_basic_page.status                           create    
             core.base_field_override.node.octagon_basic_page.promote                          create    
             node.type.page                                                                    create    
             core.base_field_override.node.page.promote                                        create    
             page_manager.page.content_with_sidebar_example_page                               create    
             page_manager.page.form_elements_example_page                                      create    
             page_manager.page.multipage_form_example_page                                     create    
             page_manager.page.node_view                                                       create    
             page_manager.page.single_column_example_page                                      create    
             page_manager.page.three_column_example_page                                       create    
             page_manager.page_variant.three_column_example_page-panels_variant-0              create    
             page_manager.page_variant.single_column_example_page-panels_variant-0             create    
             page_manager.page_variant.content_with_sidebar_example_page-panels_variant-0      create    
             field.storage.paragraph.field_octa_tst_variant                                    create    
             field.storage.paragraph.field_octa_tst_title                                      create    
             field.storage.paragraph.field_octa_tst_sub_title                                  create    
             field.storage.paragraph.field_octa_tst_link                                       create    
             field.storage.paragraph.field_octa_reference_block                                create    
             field.storage.paragraph.field_octa_node_tst_variant                               create    
             field.storage.paragraph.field_octa_node_tst_title                                 create    
             field.storage.paragraph.field_octa_node_tst_sub_title                             create    
             field.storage.paragraph.field_octa_node_tst_link                                  create    
             field.storage.paragraph.field_octa_button_text                                    create    
             field.storage.paragraph.field_octa_button_sr_message                              create    
             field.storage.paragraph.field_octa_button_link                                    create    
             field.storage.node.field_basic_page_paragraphs                                    create    
             core.entity_view_mode.paragraph.preview                                           create    
             paragraphs.paragraphs_type.octagon_block                                          create    
             field.field.paragraph.octagon_block.field_octa_reference_block                    create    
             core.entity_view_display.paragraph.octagon_block.default                          create    
             core.entity_form_display.paragraph.octagon_block.default                          create    
             paragraphs.paragraphs_type.octagon_button                                         create    
             field.field.paragraph.octagon_button.field_octa_button_text                       create    
             field.field.paragraph.octagon_button.field_octa_button_sr_message                 create    
             field.field.paragraph.octagon_button.field_octa_button_link                       create    
             core.entity_view_display.paragraph.octagon_button.default                         create    
             core.entity_form_display.paragraph.octagon_button.default                         create    
             paragraphs.paragraphs_type.octagon_node_title_sub_title                           create    
             field.field.paragraph.octagon_node_title_sub_title.field_octa_node_tst_variant    create    
             field.field.paragraph.octagon_node_title_sub_title.field_octa_node_tst_title      create    
             field.field.paragraph.octagon_node_title_sub_title.field_octa_node_tst_sub_title  create    
             field.field.paragraph.octagon_node_title_sub_title.field_octa_node_tst_link       create    
             core.entity_view_display.paragraph.octagon_node_title_sub_title.default           create    
             core.entity_form_display.paragraph.octagon_node_title_sub_title.default           create    
             paragraphs.paragraphs_type.octagon_text                                           create    
             paragraphs.paragraphs_type.octagon_title_sub_title                                create    
             field.field.paragraph.octagon_title_sub_title.field_octa_tst_variant              create    
             field.field.paragraph.octagon_title_sub_title.field_octa_tst_title                create    
             field.field.paragraph.octagon_title_sub_title.field_octa_tst_sub_title            create    
             field.field.paragraph.octagon_title_sub_title.field_octa_tst_link                 create    
             field.field.node.octagon_basic_page.field_basic_page_paragraphs                   create    
             core.entity_view_display.paragraph.octagon_title_sub_title.default                create    
             core.entity_form_display.paragraph.octagon_title_sub_title.default                create    
             core.entity_form_display.node.octagon_basic_page.default                          create    
             rdf.mapping.comment.comment                                                       create    
             rdf.mapping.node.article                                                          create    
             rdf.mapping.node.page                                                             create    
             block.block.meyers_search                                                         create    
             search.page.node_search                                                           create    
             block.block.seven_secondary_local_tasks                                           create    
             block.block.seven_primary_local_tasks                                             create    
             block.block.seven_page_title                                                      create    
             block.block.seven_local_actions                                                   create    
             block.block.seven_help                                                            create    
             block.block.seven_messages                                                        create    
             block.block.seven_content                                                         create    
             block.block.seven_breadcrumbs                                                     create    
             block.block.meyers_powered                                                        create    
             block.block.meyers_messages                                                       create    
             block.block.meyers_content                                                        create    
             block.block.meyers_breadcrumbs                                                    create    
             block.block.meyers_branding                                                       create    
             system.action.comment_publish_action                                              create    
             system.action.comment_save_action                                                 create    
             system.action.comment_unpublish_action                                            create    
             system.action.node_delete_action                                                  create    
             system.action.node_make_sticky_action                                             create    
             system.action.node_make_unsticky_action                                           create    
             system.action.node_promote_action                                                 create    
             system.action.node_publish_action                                                 create    
             system.action.node_save_action                                                    create    
             system.action.node_unpromote_action                                               create    
             system.action.node_unpublish_action                                               create    
             system.menu.account                                                               create    
             block.block.meyers_account_menu                                                   create    
             system.menu.admin                                                                 create    
             system.menu.footer                                                                create    
             block.block.meyers_footer                                                         create    
             system.menu.main                                                                  create    
             block.block.meyers_main_menu                                                      create    
             system.menu.tools                                                                 create    
             block.block.meyers_tools                                                          create    
             field.storage.node.field_tags                                                     create    
             core.entity_view_mode.taxonomy_term.full                                          create    
             taxonomy.vocabulary.tags                                                          create    
             rdf.mapping.taxonomy_term.tags                                                    create    
             field.field.node.article.field_tags                                               create    
             field.storage.paragraph.field_octa_text                                           create    
             field.storage.node.body                                                           create    
             field.storage.comment.comment_body                                                create    
             field.storage.block_content.body                                                  create    
             field.field.paragraph.octagon_text.field_octa_text                                create    
             field.field.node.page.body                                                        create    
             field.field.node.article.body                                                     create    
             field.field.comment.comment.comment_body                                          create    
             field.field.block_content.basic.body                                              create    
             core.entity_view_display.paragraph.octagon_text.default                           create    
             core.entity_view_display.comment.comment.default                                  create    
             core.entity_view_display.block_content.basic.default                              create    
             core.entity_form_display.paragraph.octagon_text.default                           create    
             core.entity_form_display.node.page.default                                        create    
             core.entity_form_display.node.article.default                                     create    
             core.entity_form_display.comment.comment.default                                  create    
             core.entity_form_display.block_content.basic.default                              create    
             system.action.user_unblock_user_action                                            create    
             system.action.user_cancel_user_action                                             create    
             system.action.user_block_user_action                                              create    
             search.page.user_search                                                           create    
             rdf.mapping.user.user                                                             create    
             field.storage.user.user_picture                                                   create    
             field.field.user.user.user_picture                                                create    
             core.entity_view_mode.user.full                                                   create    
             core.entity_view_mode.user.compact                                                create    
             core.entity_view_display.user.user.default                                        create    
             core.entity_view_display.user.user.compact                                        create    
             core.entity_view_display.node.page.teaser                                         create    
             core.entity_view_display.node.page.default                                        create    
             core.entity_view_display.node.octagon_basic_page.teaser                           create    
             core.entity_view_display.node.octagon_basic_page.default                          create    
             core.entity_view_display.node.article.teaser                                      create    
             core.entity_view_display.node.article.rss                                         create    
             core.entity_view_display.node.article.default                                     create    
             core.entity_form_mode.user.register                                               create    
             core.entity_form_display.user.user.default                                        create    
             block.block.seven_login                                                           create    
             user.role.administrator                                                           create    
             system.action.user_remove_role_action.administrator                               create    
             system.action.user_add_role_action.administrator                                  create    
             user.role.anonymous                                                               create    
             user.role.authenticated                                                           create    
             views.view.archive                                                                create    
             views.view.block_content                                                          create    
             views.view.comments_recent                                                        create    
             views.view.content                                                                create    
             views.view.content_recent                                                         create    
             views.view.files                                                                  create    
             views.view.frontpage                                                              create    
             views.view.glossary                                                               create    
             views.view.taxonomy_term                                                          create    
             views.view.user_admin_people                                                      create    
             views.view.who_s_new                                                              create    
             views.view.who_s_online                                                           create    
             system.action.webform_submission_make_unsticky_action                             create    
             system.action.webform_submission_make_sticky_action                               create    
             system.action.webform_submission_delete_action                                    create    
             page_manager.page_variant.multipage_form_example_page-panels_variant-0            create    
             page_manager.page_variant.form_elements_example_page-panels_variant-0             create    
             webform.settings                                                                  create    
             webform.webform.contact                                                           create    
             webform.webform.example_elements                                                  create    
             webform.webform.example_elements_formats                                          create    
             webform.webform.example_elements_masks                                            create    
             webform.webform.example_elements_states                                           create    
             webform.webform.example_layout_basic                                              create    
             webform.webform.example_layout_flexbox                                            create    
             webform.webform.example_options                                                   create    
             webform.webform.example_wizard                                                    create    
             webform.webform_options.country_codes                                             create    
             webform.webform_options.country_names                                             create    
             webform.webform_options.creditcard_codes                                          create    
             webform.webform_options.days                                                      create    
             webform.webform_options.education                                                 create    
             webform.webform_options.employment_status                                         create    
             webform.webform_options.ethnicity                                                 create    
             webform.webform_options.gender                                                    create    
             webform.webform_options.industry                                                  create    
             webform.webform_options.languages                                                 create    
             webform.webform_options.likert_agreement                                          create    
             webform.webform_options.likert_comparison                                         create    
             webform.webform_options.likert_importance                                         create    
             webform.webform_options.likert_quality                                            create    
             webform.webform_options.likert_satisfaction                                       create    
             webform.webform_options.likert_ten_scale                                          create    
             webform.webform_options.likert_would_you                                          create    
             webform.webform_options.marital_status                                            create    
             webform.webform_options.months                                                    create    
             webform.webform_options.relationship                                              create    
             webform.webform_options.size                                                      create    
             filter.format.basic_html                                                          create    
             webform.webform_options.state_codes                                               create    
             editor.editor.basic_html                                                          create    
             webform.webform_options.state_names                                               create    
             filter.format.full_html                                                           create    
             webform.webform_options.state_province_codes                                      create    
             config_split.config_split.dev                                                     create    
             editor.editor.full_html                                                           create    
             webform.webform_options.state_province_names                                      create    
             config_split.config_split.local                                                   create    
             contact.form.feedback                                                             create    
             filter.format.plain_text                                                          create    
             webform.webform_options.time_zones                                                create    
             config_split.config_split.prod                                                    create    
             contact.form.personal                                                             create    
             filter.format.restricted_html                                                     create    
             webform.webform_options.titles                                                    create    
             config_ignore.settings                                                            create    
             config_split.config_split.stage                                                   create    
             shortcut.set.default                                                              create    
             update.settings                                                                   create    
             webform.webform_options.yes_no                                                    create    
             core.extension                                                                    update    
             core.menu.static_menu_link_overrides                                              update    
             system.cron                                                                       update    
             system.date                                                                       update    
             system.mail                                                                       update    
             system.theme                                                                      update    
             user.settings                                                                     update    
             views.view.who_s_online                                                           delete    
             shortcut.set.default                                                              delete    
             views.view.who_s_new                                                              delete    
             filter.format.restricted_html                                                     delete    
             contact.form.personal                                                             delete    
             views.view.user_admin_people                                                      delete    
             filter.format.plain_text                                                          delete    
             contact.form.feedback                                                             delete    
             views.view.taxonomy_term                                                          delete    
             editor.editor.full_html                                                           delete    
             views.view.glossary                                                               delete    
             filter.format.full_html                                                           delete    
             views.view.frontpage                                                              delete    
             editor.editor.basic_html                                                          delete    
             views.view.files                                                                  delete    
             filter.format.basic_html                                                          delete    
             views.view.content_recent                                                         delete    
             views.view.content                                                                delete    
             views.view.comments_recent                                                        delete    
             views.view.block_content                                                          delete    
             views.view.archive                                                                delete    
             user.role.authenticated                                                           delete    
             user.role.anonymous                                                               delete    
             system.action.user_add_role_action.administrator                                  delete    
             system.action.user_remove_role_action.administrator                               delete    
             user.role.administrator                                                           delete    
             block.block.seven_login                                                           delete    
             core.entity_form_display.user.user.default                                        delete    
             core.entity_form_mode.user.register                                               delete    
             core.entity_view_display.node.article.default                                     delete    
             core.entity_view_display.node.article.rss                                         delete    
             core.entity_view_display.node.article.teaser                                      delete    
             core.entity_view_display.node.page.default                                        delete    
             core.entity_view_display.node.page.teaser                                         delete    
             core.entity_view_display.user.user.compact                                        delete    
             core.entity_view_display.user.user.default                                        delete    
             core.entity_view_mode.user.compact                                                delete    
             core.entity_view_mode.user.full                                                   delete    
             field.field.user.user.user_picture                                                delete    
             field.storage.user.user_picture                                                   delete    
             rdf.mapping.user.user                                                             delete    
             search.page.user_search                                                           delete    
             system.action.user_block_user_action                                              delete    
             system.action.user_cancel_user_action                                             delete    
             system.action.user_unblock_user_action                                            delete    
             core.entity_form_display.block_content.basic.default                              delete    
             core.entity_form_display.comment.comment.default                                  delete    
             core.entity_form_display.node.article.default                                     delete    
             core.entity_form_display.node.page.default                                        delete    
             core.entity_view_display.block_content.basic.default                              delete    
             core.entity_view_display.comment.comment.default                                  delete    
             field.field.block_content.basic.body                                              delete    
             field.field.comment.comment.comment_body                                          delete    
             field.field.node.article.body                                                     delete    
             field.field.node.page.body                                                        delete    
             field.storage.block_content.body                                                  delete    
             field.storage.comment.comment_body                                                delete    
             field.storage.node.body                                                           delete    
             field.field.node.article.field_tags                                               delete    
             rdf.mapping.taxonomy_term.tags                                                    delete    
             taxonomy.vocabulary.tags                                                          delete    
             core.entity_view_mode.taxonomy_term.full                                          delete    
             field.storage.node.field_tags                                                     delete    
             system.menu.tools                                                                 delete    
             system.menu.main                                                                  delete    
             system.menu.footer                                                                delete    
             system.menu.admin                                                                 delete    
             system.menu.account                                                               delete    
             system.action.node_unpublish_action                                               delete    
             system.action.node_unpromote_action                                               delete    
             system.action.node_save_action                                                    delete    
             system.action.node_publish_action                                                 delete    
             system.action.node_promote_action                                                 delete    
             system.action.node_make_unsticky_action                                           delete    
             system.action.node_make_sticky_action                                             delete    
             system.action.node_delete_action                                                  delete    
             system.action.comment_unpublish_action                                            delete    
             system.action.comment_save_action                                                 delete    
             system.action.comment_publish_action                                              delete    
             block.block.seven_breadcrumbs                                                     delete    
             block.block.seven_content                                                         delete    
             block.block.seven_messages                                                        delete    
             block.block.seven_help                                                            delete    
             block.block.seven_local_actions                                                   delete    
             block.block.seven_page_title                                                      delete    
             block.block.seven_primary_local_tasks                                             delete    
             block.block.seven_secondary_local_tasks                                           delete    
             search.page.node_search                                                           delete    
             rdf.mapping.node.page                                                             delete    
             rdf.mapping.node.article                                                          delete    
             rdf.mapping.comment.comment                                                       delete    
             core.base_field_override.node.page.promote                                        delete    
             node.type.page                                                                    delete    
             field.field.node.article.comment                                                  delete    
             field.field.node.article.field_image                                              delete    
             node.type.article                                                                 delete    
             core.entity_view_mode.node.full                                                   delete    
             core.entity_view_mode.node.rss                                                    delete    
             core.entity_view_mode.node.search_index                                           delete    
             core.entity_view_mode.node.search_result                                          delete    
             core.entity_view_mode.node.teaser                                                 delete    
             field.storage.node.comment                                                        delete    
             field.storage.node.field_image                                                    delete    
             image.style.thumbnail                                                             delete    
             image.style.medium                                                                delete    
             image.style.large                                                                 delete    
             core.entity_view_mode.comment.full                                                delete    
             core.entity_view_mode.block_content.full                                          delete    
             core.date_format.short                                                            delete    
             core.date_format.medium                                                           delete    
             core.date_format.long                                                             delete    
             core.date_format.html_yearless_date                                               delete    
             core.date_format.html_year                                                        delete    
             core.date_format.html_week                                                        delete    
             core.date_format.html_time                                                        delete    
             core.date_format.html_month                                                       delete    
             core.date_format.html_datetime                                                    delete    
             core.date_format.html_date                                                        delete    
             core.date_format.fallback                                                         delete    
             comment.type.comment                                                              delete    
             block_content.type.basic                                                          delete    
             tour.tour.views-ui                                                                delete    
             field_ui.settings                                                                 delete    
             dblog.settings                                                                    delete    
             block.block.bartik_tools                                                          delete    
             block.block.bartik_main_menu                                                      delete    
             block.block.bartik_footer                                                         delete    
             block.block.bartik_account_menu                                                   delete    
             block.block.bartik_branding                                                       delete    
             block.block.bartik_breadcrumbs                                                    delete    
             block.block.bartik_content                                                        delete    
             block.block.bartik_messages                                                       delete    
             block.block.bartik_powered                                                        delete    
             block.block.bartik_search                                                         delete    
             block.block.bartik_help                                                           delete    
             block.block.bartik_page_title                                                     delete    
             block.block.bartik_local_tasks                                                    delete    
             block.block.bartik_local_actions                                                  delete
Import the listed configuration changes? (y/n): y
Drupal\Core\Config\ConfigImporterException: There were errors validating the config synchronization. in Drupal\Core\Config\ConfigImporter->validate() (line 728 of                        [error]
/var/www/octagon/docroot/core/lib/Drupal/Core/Config/ConfigImporter.php).
The import failed due for the following reasons:                                                                                                                                          [error]
Entities exist of type <em class="placeholder">Shortcut link</em> and <em class="placeholder"></em> <em class="placeholder">Default</em>. These entities need to be deleted before
importing.
[phingcall] /Users/stephanie/websites/octagon/./vendor/acquia/blt/phing/tasks/drupal.xml:15:10: /Users/stephanie/websites/octagon/./vendor/acquia/blt/phing/tasks/drupal.xml:32:44: Drush exited with code 1

BUILD FAILED/Users/stephanie/websites/octagon/phing/tasks/init-site.xml:73:49: Execution of the target buildfile failed. Aborting.
; 51.2642 seconds

```

And I expected this to happen:
I expected the config import to complete without any build failed messages. 

Based on this issue i found: https://www.drupal.org/node/2638392#comment-11816155 , I attempted to edit `/config/niche/shortcut.set.default.yml` and set the UUID to null. After doing so and re-importing config manually with `drush @site-alias cim`, that seemed to work. 
However, when i try to run the setup command again, it seems like Drupal installs the shortcut module once more and attempts to import the incorrect shortcut module's config from the Drupal install config files.

**Comment by bobbygryzynger** [comment]
The issue here ended up being that `cm.core.dirs.sync.path` is always set to: `${cm.core.path}/default`, setting this to `${cm.core.path}/${multisite.name}` in `project.yml` resolved this problem for us as this was any attempt to run `setup:drupal:install` on a non-`default` site with its own configuration. This issue can be closed.

**Comment by leonk** [comment]
We're having the same issue, but we're not using multisite, so I don't see how to resolve it (everything has to use the `default` directory).

We're using the standard profile, I just set this up on a fresh site.  Exported my configuration to the default folder. My shortcut.set.default.yml file contained a uuid.
Now if I run `blt setup` again, I get the error from the post above (`Entities exist of type <em class="placeholder">Shortcut link</em> and <em class="placeholder"></em> <em class="placeholder">Default</em>. These entities need to be deleted before
importing.`)
If I set it to `uuid: null` and re-run `blt setup` it gets passed the point of doing the drupal install, but fails when it tries to import the configuration again (due to mismatching uuid's).

Basically, I think running the standard profile with BLT is currently broken.

**Comment by arisaves** [comment]
Wondering if this is ever going to be addressed; I don't feel like we should be forced to use the lightning profile to avoid this. Really struggling with this, results for fixes are inconsistent, constant re-work, any help would be appreciated.

**Comment by danepowell** [comment]
I don't know that there's much BLT can do to address this, it's a Drupal core problem and can be easily reproduced by just using Drush to install the Standard profile and import configuration. For a good summary of the problem and quick fixes, see https://github.com/acquia/reservoir/issues/47

For a somewhat more in-depth explanation and fixes see https://danepowell.com/blog/installing-sites-existing-config-drupal-8
